pkg/meta/model/job.go

```diff
@@ -733,6 +748,10 @@ func (job *Job) IsPausing() bool {
 
 // IsPausable checks whether we can pause the job.
 func (job *Job) IsPausable() bool {
+	// TODO: We can remove it after TiFlash support pause operation.
+	if job.Type == ActionAddVectorIndex && job.SchemaState == StateWriteReorganization {
+		return false
+	}
```

Comment:
Maybe put this to `job.IsRollbackable()`

Reply:
But this state supports cancel and rollback, we just don't support pause at the moment. So I put it here.